symengine/cwrapper.cpp

```diff
@@ -1,4 +1,5 @@
-#include <stdlib.h>
+#include <cstdlib>
+#include <cstring>
```

Comment:
Some compilers will simply use this header when `std::string` is used, but I had to add this explicitly otherwise, the compiler complains that `error: ‘strcpy’ is not a member of ‘std’`